src/addtodo.js

```diff
@@ -0,0 +1,71 @@
+const listParent = document.querySelector('.list');
+const returnIcon = document.querySelector('.add span');
+const textInput = document.querySelector('.add input');
+
+let tasks = localStorage.getItem('todos')
+  ? JSON.parse(localStorage.getItem('todos'))
+  : [];
+
+const updateTasks = () => {
+  listParent.innerHTML = '';
+  tasks.forEach((task) => {
+    listParent.innerHTML += `
+   <li draggable ="true" id="${task.index}">
+     <div class="content">
+       <input class="check" type="checkbox" ${task.completed ? 'checked' : ''}/>
+       <input class="input" type="text" value='${task.description}' readonly />
+     </div>
+     <div class="actions">
+       <span class="material-icons drag">more_vert</span>
+       <span class="material-icons" id="deleteicon">delete_outline</span>
+     </div>
+   </li>
+   `;
+  });
+
+  const deletButton = document.querySelectorAll('#deleteicon');
+  deletButton.forEach((x) => {
+    x.addEventListener('click', () => {
+      const id = Number(x.parentNode.parentNode.id);
+      tasks = tasks.filter((task) => task.index !== id);
```

Comment:
Your **app is not functioning as intended** and therefore needs further improvement. This is likely due to improperly implementing changes requested by the previous code reviewer.
- [ ] Entries are not being stored in the local storage properly. When I add the first entry as _one_ and then the _second_ entry as _two_, local storage modifies both entries as _two_, and _twos_ is displayed twice instead of _one_ and _two_. Kindly improve upon this section.
- [ ] With regards to the above also ensure that new entries **match** the property index. The _property index_ must be set to the value of the _new array length_ (i.e. if a 5th task is added to the list, the index of that task should equal 5).

**Error with storage and mismatch index**
![image](https://user-images.githubusercontent.com/12745474/149301804-8abebe08-756a-4d9c-8c1f-5e12b7977b74.png)